applications/asset_tracker/CMakePresets.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": 1,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 19,
+    "patch": 0
```

Comment:
What I can advise from my experience with presets is to use possibly the newest Preset manifest version.
Looks like presets for CMake configure, build and test steps were added as separate steps in CMake v3.20.0 (manifest ver 2.0). Features grow very dynamically and we will need them to group configurations using inheritance successfully.  Zephyr minimal Cmake is 3.20.0 as far as I remember?